scripts/known_good.json

```diff
@@ -14,7 +14,7 @@
       "sub_dir" : "MoltenVK",
       "build_dir" : "MoltenVK",
       "install_dir" : "MoltenVK",
-      "commit" : "v1.2.3",
+      "commit" : "v1.2.4",
```

Comment:
Technically, this is a `tag`, not a `commit`... but it's no worse than its predecessor.